index.html

```diff
@@ -15,6 +15,15 @@
     const portal = new Portal(sites);
     portal.install(document.body);
     portal.start();
+
+    function toggleType(e) {
```

Comment:
Are you able to move this out of index.hml and into portal.js? Then, just change the onClick handers to be `portal.toggleType(event)`.

Reply:
sure

Reply:
👍